larray/core/session.py

```diff
@@ -1150,9 +1231,15 @@ def summary(self, template=None):
             template[Group] = "{key}: {axis_name}{labels} >> {name} ({length})"
         if LArray not in template:
             template[LArray] = "{key}: {axes_names} ({shape}) [{dtype}]\n    {title}"
+        if Metadata not in template:
+            template[Metadata] = "{key} (metadata): {value}"
 
         def display(k, v):
-            t = Group if isinstance(v, Group) else type(v)
+            if not isinstance(v, (Axis, Group, LArray)):
```

Comment:
I would **much** rather have a single template for Metadata objects, so that users can chose to indent it more than the rest or not to display it at all. It would also (re-) allow user-defined objects in the Session.